packages/ui/components/toast/showToast.tsx

```diff
@@ -72,28 +73,28 @@ export function showToast(
   variant: "success" | "warning" | "error",
   duration = TOAST_VISIBLE_DURATION
 ) {
-  const onClose = () => {
-    toast.dismiss();
+  const onClose = (toastId: string) => {
+    toast.remove(toastId);
```

Comment:
Do we need to switch to remove ? I think we should use dismiss only as using `remove` would cause dismiss animation to not work.

Reply:
I think we don't need a delay to remove the toaster. If someone wants to remove a toaster, then he/she wants to remove it quickly with a click. What do you think?

Reply:
I agree with this statement, removing a toast is something that should happen asap, an animation isn't needed.